crypto/src/main/java/org/hyperledger/besu/crypto/SignatureAlgorithm.java

```diff
@@ -54,6 +55,8 @@ SECPSignature normaliseSignature(
 
   String getCurveName();
 
+  DSAKCalculator getKCalculator();
```

Comment:
I think this function does not have to be part of the interface, as it only used internally.

Reply:
Good point. I've removed it from the interface! 👍